crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -27,6 +31,56 @@ fn convert_level(level: LogLevel) -> Level {
     }
 }
 
+#[wasm_bindgen]
+pub struct BitwardenPure;
+
+#[wasm_bindgen]
+impl BitwardenPure {
+    pub fn version() -> String {
+        Self::setup_once();
+        env!("SDK_VERSION").to_owned()
+    }
+
+    pub fn echo(msg: String) -> String {
+        Self::setup_once();
+        msg
+    }
+
+    pub fn throw(msg: String) -> Result<(), TestError> {
+        Self::setup_once();
+        Err(TestError(msg))
+    }
+
+    pub fn symmetric_decrypt(
+        enc_string: String,
+        key_b64: String,
+    ) -> Result<String, PureCryptoError> {
+        Self::setup_once();
+        pure_crypto::symmetric_decrypt(enc_string, key_b64)
+    }
+
+    pub fn symmetric_decrypt_to_bytes(
```

Comment:
I think we want an additional function here that decrypts the enc buffers (attachments)